src/main/java/io/logz/logback/LogzioSender.java

```diff
@@ -307,6 +323,23 @@ private String formatMessage(ILoggingEvent loggingEvent) {
         logMessage.addProperty("logger", loggingEvent.getLoggerName());
         logMessage.addProperty("thread", loggingEvent.getThreadName());
 
+        if (hostname != null) {
+            logMessage.addProperty("hostname", hostname);
+        }
+        if (additionalFields != null) {
+            additionalFields.forEach((k,v) -> {
+                if (v.startsWith("$")) {
```

Comment:
If you only support complete value and not "hello $USER there", you need to specify that in the documentation